audittail-helm-library/tests/consumerapp/templates/deployment.yaml

```diff
@@ -21,6 +16,9 @@ spec:
     spec:
       initContainers:
         {{- include "audittail.initContainer" .| nindent 8 }}
+          securityContext:
+            allowPriviligesEsclation: false
+            runAsNonroot: true
```

Comment:
I rather have these be part of our defaults and not something a user has to set up. Could we move this?

Reply:
Updated as requested